app/controllers/ember-users.js

```diff
@@ -1,10 +1,14 @@
 import Controller from '@ember/controller';
-import { sort } from '@ember/object/computed';
 import { inject as service } from '@ember/service';
 
-export default Controller.extend({
-  fastboot: service(),
+export default class EmberUsers extends Controller {
+  @service fastboot;
 
-  sortingKey: Object.freeze(['featured:desc', 'added']),
-  sortedModel: sort('model', 'sortingKey'),
-});
+  get sortedModel() {
+    return this.model.toArray().sort((a, b) => {
+      let featuredComp = (a.featured < b.featured) * 2 - 1;
+      let addedComp = a.added - b.added;
+      return -featuredComp || addedComp; // featured DESC, added ASC
+    });
+  }
```

Comment:
I agree that we can use JavaScript's `.sort()` to sort the data. However, to keep the pull request's scope small for review, can you use `@sort` instead?

```suggestion
  sortingKey = Object.freeze(['featured:desc', 'added']);
  /* eslint-disable-next-line ... */
  @sort('model', 'sortingKey') sortedModel;
```

I think making fewer changes in one pull request will help with tests passing.